### ISSUE TITLE
adding code of conduct info to handbook

### ISSUE THREAD

**Issue by ChristinaLK** [carpentries/handbook#218]
There have been at least three threads in the instructor training repo that I think would be relevant for the handbook: 
- how to introduce the Code of Conduct on workshops: https://github.com/carpentries/instructor-training/issues/645
- when introducing the CoC, identifying if you're a mandated reporter (https://github.com/carpentries/instructor-training/issues/659)
- how to respond to incidents (https://github.com/carpentries/instructor-training/issues/690)

Can/should these be added to the handbook?  If so, where should each go?  
I can put them in the instructor training materials, but that feels a little "buried".

**Comment by maneesha** [comment]
pinging @kariljordan  and @ErinBecker  on this

**Comment by kariljordan** [comment]
@karinlag and the CoC committee are currently revising the CoC. Once that's complete we'll update the handbook.

**Comment by maneesha** [comment]
Assigning to @kariljordan  for whenever the CoC committee has completed revisions to the handbook.

**Comment by kariljordan** [comment]
Thanks! Per @karinlag, the CoC is currently discussing a RfC process. I put a note in my Asana to revise this when the CoC has been updated.

**Comment by kariljordan** [comment]
I made a note to come back to this issue on March 1st after the revised CoC documentation is up.

**Comment by kariljordan** [comment]
> There have been at least three threads in the instructor training repo that I think would be relevant for the handbook:
> 
> * how to introduce the Code of Conduct on workshops: [carpentries/instructor-training#645](https://github.com/carpentries/instructor-training/issues/645)

I think the "how to" should remain in the instructor training material. The Code of Conduct is under the **Policies** section in the handbook, so we want to keep the language focused on the policy. I am open to a debate though! 

> * when introducing the CoC, identifying if you're a mandated reporter ([carpentries/instructor-training#659](https://github.com/carpentries/instructor-training/issues/659))

This came up on a Slack thread, and @elliewix  mentioned in the CoC repo that it's a good idea that could apply to most groups, but not all. Let's make this a separate issue and ask someone to submit a PR to recommend the phrasing.
> * how to respond to incidents ([carpentries/instructor-training#690](https://github.com/carpentries/instructor-training/issues/690))

The handbook now includes [Incident Response Guidelines](https://docs.carpentries.org/topic_folders/policies/incident-response.html)

> Can/should these be added to the handbook? If so, where should each go?
> I can put them in the instructor training materials, but that feels a little "buried".

In summary, I'm recommending we keep the "how to talk about the CoC" in instructor training and file an issue specifically for the mandated reporter language. The handbook includes incident response guidelines.

**Comment by maneesha** [comment]
@kariljordan  Is this ok to close now?

**Comment by kariljordan** [comment]
I'm fine to close this, but @ChristinaLK opened the issue, so let's make sure she's happy with my response.

**Comment by ChristinaLK** [comment]
Yes, looks good to me!  Thanks @kariljordan !
